docs/tutorial/chapter-2-writing-your-first-schema.md

```diff
@@ -140,13 +140,30 @@ type Query {
 
-Now, the `resolve` property is where you, the developer, implement this process of getting data. Put another way, the `resolve` property is where you implement the logic that fulfills the field's specification. You may now be noting how when we defined our `User` object we did _not_ write resolvers for its fields. The reason for that is that Nexus provides _default_ resolvers for fields that are scalars and that are not roots. This default implementation is to return a property from the parent data of the same name as the field name. And when that's not possible (because the parent data diverges), then Nexus will let you know _statically_, requiring the resolver from you. What's awesome is that none of this is require knowledge to get productive with Nexus thanks to the static error that Nexus will give you along the way right in your IDE. Follow these and for the most part you'll fall into the pit of success. Aweomse!
+Now, the `resolve` property is where you, the developer, implement this process of getting data. Put another way, the `resolve` property is where you implement the logic that fulfills the field's specification. You may be noting how when we defined our `User` object, we did _not_ write resolvers for its fields. The reason for that is that Nexus provides _default_ resolvers for fields that are not root and that don't have resolvers. This default implementation is to return a property from the parent data of the same name as the field name. And when that's not possible (because the parent data diverges), then Nexus will let you know _statically_, requiring the resolver from you. What's awesome is that none of this is require knowledge to get productive with Nexus thanks to the static error that Nexus will give you along the way right in your IDE. Follow these and for the most part you'll fall into the pit of success. Awesome!
 
 We will not go into more detail about the data resolution systems of GraphQL and Nexus just now. This was just a brief overview to give you a sense of what is going on. Mastering a complete mental model will take a bit of time and practice.
 
-So go ahead and add an empty `resolve` method now. Once done, you will see a new error from your IDE about the return type of `resolve`. This makes sense. Our resolver, being empty, is not satisfying the specification we gave for `Query.products`. Thanks Nexus! Play around with autocompletion to return a value that will fix the error.
+So go ahead and add an empty `resolve` method now. Once done, you will see a new error from your IDE about the return type of `resolve`. This makes sense. Our resolver, being empty, is not satisfying the specification we gave for `Query.users`. Thanks Nexus! Play around with autocompletion to return a value that will fix the error.
+
+```ts
```

Comment:
Should we make this one a diff? It's semi consistent to do with other flows in the tutorial.

One thing I tried to do was reduce the overall duplication in favour of upfront code blocks and then a few diffs to adjust/build up. The tradeoff here is that, I feel, its easier to scroll and around and anchor oneself but at any given point you cannot always see "what it looks like now". I wanted to solve the tradeoff by using tabs but couldn't get them to work in docsify. I guess we'll wait for the new website for tabs. Once we have those, we can have diffs in the first tab and a running "total state" secondary tab.

0.02

Reply:
Let's do it 👍